test/e2e/framework/fips.go

```diff
@@ -0,0 +1,89 @@
+/*
+SPDX-License-Identifier: Apache-2.0
+
+Copyright Contributors to the Submariner project.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package framework
+
+import (
+	"context"
+	"fmt"
+	"strings"
+
+	. "github.com/onsi/gomega"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+const (
+	fipsNamespace     = "kube-system"
+	fipsConfigMapName = "cluster-config-v1"
+)
+
+func FindFipsConfig(cluster ClusterIndex) bool {
+	configMap, err := KubeClients[cluster].CoreV1().ConfigMaps(fipsNamespace).Get(context.TODO(), fipsConfigMapName, metav1.GetOptions{})
+	if err != nil {
+		if strings.Contains(err.Error(), "not found") {
```

Comment:
Use `IsNotFound` in `k8s.io/apimachinery/pkg/api/errors`:

```suggestion
		if apierrors.IsNotFound(err) {
```

(importing `k8s.io/apimachinery/pkg/api/errors` as `apierrors`).

Reply:
Done